contrib/cirrus/runner.sh

```diff
@@ -77,6 +77,7 @@ _run_setup() {
     fi
     msg "Copying test binaries from $SKOPEO_CIDEV_CONTAINER_FQIN /usr/local/bin/"
     cp -a "$mnt/usr/local/bin/"* "/usr/local/bin/"
+    cp "$mnt/usr/bin/registry" "/usr/local/bin/"
```

Comment:
The tests are _supposed_ to be looking in `$PATH`. I’m confused about how that is failing.